src/ProjectOrigin.VerifiableEventStore.Tests/MemoryEventStoreTests.cs

```diff
@@ -0,0 +1,33 @@
+using ProjectOrigin.VerifiableEventStore.Models;
+using ProjectOrigin.VerifiableEventStore.Services.EventStore;
+
+namespace ProjectOrigin.VerifiableEventStore.Tests;
+
+public class MemoryEventStoreTests
+{
+    [Fact]
+    public async Task MemoryEventStore_StoreEvents_ReturnsBatch()
+    {
+        var batch1 = new Fixture().Create<Batch>();
+        var batch2 = new Fixture().Create<Batch>();
+
+        var meroryEventStore = new MemoryEventStore();
+
+        await meroryEventStore.StoreBatch(batch1);
+        await meroryEventStore.StoreBatch(batch2);
+
+        var batchResult = await meroryEventStore.GetBatch(batch1.Events.First().Id);
```

Comment:
Looks like a typo here meroryEventStore.... should probably be memoryEventStore...